### PR TITLE
chore(renovate): Switch to monthly merge

### PR DESCRIPTION
**Pull request by KristjanESPERANTO**
I think a weekly merge would be better. Now the users can get daily update notifications.

### CODE REVIEW

**Comment by CFenner** [comment]
Thanks for looking into this. I also wanted to change it, but haven't had time yet. Maybe even change it to monthly. What do you think?

File: .github/renovate.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "config:base",
-    "schedule:automergeDaily",
+    "schedule:earlyMondays",
```
Comment (CFenner) [inline .github/renovate.json:5]: keep auto merge

```suggestion
    "schedule:automergeEarlyMondays",
```

File: .github/renovate.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "config:base",
-    "schedule:automergeDaily",
+    "schedule:automergeEarlyMondays",
```
Comment (CFenner) [inline .github/renovate.json:5]:
```suggestion
    "schedule:automergeMonthly",
```

Comment (KristjanESPERANTO) [reply .github/renovate.json:5]: Yes, monthly is certainly often enough :+1: 🙂

**Pull request merged**